File: 10.js
const readline = require("readline");
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
});
rl.question("Enter a year: ", function (input) {
    const year = parseInt(input)
    if ((year % 4 === 0 && year % 100 !== 0) || (year % 400 === 0)) {
        console.log(`${year} \nis a leap year`);
    } else {
        console.log(`${year} \nis not a leap year`);
    }
    rl.close();
});

File: 10.cpp
#include <iostream>
using namespace std;

int main() 
{
    int year;
    cout << "Enter a year: ";
    cin >> year;
    if ((year % 4 == 0 && year % 100 != 0) || (year % 400 == 0))
    {
        cout << year << " \nleap year" << endl;
    } 
    else 
    {
        cout << year << "\nnot a leap year" << endl;
    }
    return 0;
}